controllers/replicationdestination.go

```diff
@@ -347,6 +347,9 @@ func (r *VolumeSnapshotRestoreReconciler) configureRepDestResticVolOptions(vsr *
 		}
 	}
 
+	repDestResticVolOptions.CustomCA.Key = "RESTIC_CUSTOM_CA"
+	repDestResticVolOptions.CustomCA.SecretName = resticSecretName
```

Comment:
Perhaps we shouldn't be setting this if customca isn't defined.

Reply:
Yes, I did more testing and volsync does not work when the secret doesn't actually have this set. I will make this open the secret and check first.

Reply:
Okay, that's working better now.